eth/stagedsync/stage_snapshots.go

```diff
@@ -418,6 +430,12 @@ func FillDBFromSnapshots(logPrefix string, ctx context.Context, tx kv.RwTx, dirs
 	return nil
 }
 
+func computeBlocksToPrune(cfg SnapshotsCfg) (blocksToPrune uint64, historyToPrune uint64) {
```

Comment:
let's rename this func to `canPruneFilesTo` - because it rely on `.SegmentsMax()` - and it's only about "pruning files" not "pruning data in db".